lib/persona-ruby/actions/inquiries.rb

```diff
@@ -22,6 +22,10 @@ def inquiry_update(id, attributes = {})
       def inquiry_delete(id)
         connection.delete("inquiries/#{id}", {})
       end
+
+      def inquiry_one_time_link(id, options = {})
```

Comment:
I left options for more custom possibilities in the future, and persona already implement a default value in api side